layouts/default.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <v-app dark>
+  <v-app ref="app" dark>
```

Comment:
~I don't think the 'dark' is needed since it should follow the vuetify color scheme.~ 

Dark isn't a prop on v-app https://vuetifyjs.com/en/components/application/